scripts/binaryninja/README.md

```diff
@@ -0,0 +1,18 @@
+To use a plugin:
+
+1. Symlink the desired plugin into the directory specified here: https://github.com/Vector35/binaryninja-api/tree/dev/python/examples#loading-plugins
```

Comment:
Perhaps it is cleaner to have an inline link with 

[into the BinaryNinja plugin directory](https://github.com/Vector35/binaryninja-api/tree/dev/python/examples#loading-plugins)